scripts/utility/fabric_images.sh

```diff
@@ -4,35 +4,17 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
-set -o pipefail
+set -euo pipefail
 
-echo "======== PULL DOCKER IMAGES ========"
+version=2.1
+artifactory_url=hyperledger-fabric.jfrog.io
 
-##########################################################
-# Pull and Tag the fabric and fabric-ca images from Artifactory
-##########################################################
-echo "Fetching images from Artifactory"
-ARTIFACTORY_URL=hyperledger-fabric.jfrog.io
+for image in peer orderer ccenv baseos nodeenv javaenv tools; do
+    artifactory_image="${artifactory_url}/fabric-${image}:amd64-${version}-stable"
+    docker pull "${artifactory_image}"
+    docker tag "${artifactory_image}" "hyperledger/fabric-${image}"
+    docker rmi -f "${artifactory_image}"
+done
 
-ARCH=amd64
-STABLE_VERSION=2.1-stable
-STABLE_TAG="${ARCH}-${STABLE_VERSION}"
-VERSION_TAG="${STABLE_VERSION%%-*}"
-echo "---------> STABLE_VERSION: ${STABLE_VERSION}"
-
-dockerTag() {
-    for IMAGE in ca peer orderer ccenv baseos nodeenv javaenv tools; do
-        ARTIFACTORY_IMAGE="${ARTIFACTORY_URL}/fabric-${IMAGE}:${STABLE_TAG}"
-        IMAGE_NAME="fabric-${IMAGE}"
-        docker pull "${ARTIFACTORY_IMAGE}"
-        docker tag "${ARTIFACTORY_IMAGE}" "hyperledger/${IMAGE_NAME}"
-        docker tag "${ARTIFACTORY_IMAGE}" "hyperledger/${IMAGE_NAME}:${VERSION_TAG}"
-        docker rmi -f "${ARTIFACTORY_IMAGE}"
-    done
-}
-
-dockerTag
-
-echo
-docker images | grep "hyperledger*"
```

Comment:
Can we have the 'docker images' here so that we have log entry of what we are running

Reply:
It won't actually help, we pull the 2.1 images, but retag them as latest, so all you see is the latest tag for everything. What I did to make up for this was add the `-q` flag to the pull script, which silences the output, so you get really clean output on what images were actually pulled, instead of all of the docker download progress crap you normally see, this is what it looks like now:

```
hyperledger-fabric.jfrog.io/fabric-peer:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-peer:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-peer@sha256:3e1b9cd6aef2c324db4f15d92f7b168bfe3ea47499735127a525fc24c39e8351
hyperledger-fabric.jfrog.io/fabric-orderer:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-orderer:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-orderer@sha256:30ea6c16de34f77b3cbcab2b1420559b1f546cfc00f10388f8d088b7b40db64a
hyperledger-fabric.jfrog.io/fabric-tools:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-tools:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-tools@sha256:6a857e11e3354f4286633d41f8e669b7390d6202ecd61d400821c105284ca337
hyperledger-fabric.jfrog.io/fabric-ccenv:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-ccenv:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-ccenv@sha256:135878ca7b2001075f814caa5d4ef7069fc25dcfd53bc4d4a1499d97c5a62d0e
hyperledger-fabric.jfrog.io/fabric-javaenv:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-javaenv:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-javaenv@sha256:2d833489f14e280d22f2420dbf87a67ee2a54ed8c082fa93cf54c49da4df3d7b
hyperledger-fabric.jfrog.io/fabric-nodeenv:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-nodeenv:amd64-2.1-stable
Untagged: hyperledger-fabric.jfrog.io/fabric-nodeenv@sha256:c3be129835bc048c9a59f2772cc0d5c3b19e52d52494e809935133c4cb25f2c1
docker.io/hyperledger/fabric-ca:1.4
```

Reply:
And the output only shows if the image was successfully pulled, if something else happens the entire process fails with the actual error output

Reply:
great